build.gradle

```diff
@@ -1,53 +1,39 @@
 apply plugin: 'idea'
 
-version '1.0.0-SNAPSHOT'
-
 subprojects {
-    apply plugin: 'idea'
-    apply plugin: 'java'
-//    apply plugin: 'checkstyle'
-//    apply plugin: 'findbugs'
-//    apply plugin: 'jacoco'
-//    apply plugin: 'pmd'
+
+    buildscript {
+        repositories {
+            mavenCentral()
+            mavenLocal()
+            jcenter()
+            maven {
+                url "https://plugins.gradle.org/m2/"
+            }
+        }
+
+        dependencies {
+            classpath "org.springframework.boot:spring-boot-gradle-plugin:${version_spring_boot}"
+            classpath "org.asciidoctor:asciidoctor-gradle-plugin:1.5.3"
+            classpath "org.ajoberstar:gradle-git:1.5.1"
+            classpath "com.diffplug.spotless:spotless-plugin-gradle:3.0.0"
+            classpath "com.github.jacobono:gradle-jaxb-plugin:1.3.6"
+        }
+    }
+
+    apply plugin: "idea"
+    apply plugin: "java"
+
+    version "1.0.0-SNAPSHOT"
 
     sourceCompatibility = 1.8
+    targetCompatibility = 1.8
 
     repositories {
         mavenCentral()
         mavenLocal()
     }
 
-//    findbugs{
-//        sourceSets = [sourceSets.main]
-//        ignoreFailures = true
-//    }
-//
-//    jacocoTestReport{
-//        reports{
-//            xml{
-//                enabled true
-//            }
-//            html{
-//                enabled true
-//            }
-//        }
-//    }
-//
-//    checkstyle {
-//        toolVersion = '7.0'
-//    }
-
-}
-// this task has been automatically added by coderadar to access all JAR dependencies for bytecode analysis
-task printClasspathForCoderadar {
-    doLast {
-        configurations.testRuntime.each { println it }
-    }
-}
-
-task wrapper(type: Wrapper) {
```

Comment:
Each time you call `gradlew wrapper` the wrapper is updated by the definition from the `build.gradle`. So you can define which wrapper version should be installed in your project without updating the `gradle-wrapper.properties` by hand.